src/accelerate/utils/imports.py

```diff
@@ -191,3 +200,22 @@ def get_major_and_minor_from_version(full_version):
         )
         return False
     return True
+
+@lru_cache()
+def is_xpu_available(check_device=True):
+    "Checks if `intel_extension_for_pytorch` is installed and potentially if a XPU is in the environment"
+    if is_ipex_available():
+        import torch
+        if version.parse(version.parse(torch.__version__).base_version) < version.parse("1.13"):
+            return False
+    else:
+        return False
```

Comment:
Let's use importlib here to have the check with `check_device=False` be cheap to execute.

Reply:
Ok there is already a ```is_torch_version``` in versions.py which invokes the importlib which can be used.